Sources/FigmaExport/Input/Params.swift

```diff
@@ -105,6 +105,7 @@ struct Params: Decodable {
         let xcassetsPath: URL
         let xcassetsInMainBundle: Bool
         let xcassetsInSwiftPackage: Bool?
+        let resourceBundleNames: [String]
```

Comment:
This parameter must be optional to support backward compatibility. If this parameter will be required, current FigmaExport users will need to specify a value to this property. Moreover a lot of users don't store xcassets in the Swift Package and won't specify a value for this property.